lib/plugins/encrypted_parameters/kms_key.rb

```diff
@@ -11,12 +13,27 @@ def initialize(arn)
         end
 
         def self.create
-          resp = Aws::KMS::Client.new.create_key
+          standard_tags=KmsKey.stack_tags
```

Comment:
same, class method can be directly called.
```suggestion
          standard_tags = stack_tags
```